mlir/include/mlir-c/Target/ExportSMTLIB.h

```diff
@@ -21,9 +21,13 @@ extern "C" {
 
 /// Emits SMTLIB for the specified module using the provided callback and user
 /// data
-MLIR_CAPI_EXPORTED MlirLogicalResult mlirExportSMTLIB(MlirModule,
-                                                      MlirStringCallback,
-                                                      void *userData);
+MLIR_CAPI_EXPORTED MlirLogicalResult
+mlirTranslateModuleSMTLIB(MlirModule, MlirStringCallback, void *userData,
```

Comment:
```suggestion
mlirTranslateModuleToSMTLIB(MlirModule, MlirStringCallback, void *userData,
```
To stay consistent with `mlirTranslateOperationToSMTLIB`?